app/modules/intelligence/tools/hypothesis_state_tool.py

```
@@ -0,0 +1,324 @@
+        rec = HypothesisRecord(
+            id=self._next_id(),
+            title=title,
+            status=status,
+            evidence=list(evidence),
+            validation_plan=list(validation_plan),
+            created_at=now,
+            updated_at=now,
+            conversation_id=self.conversation_id,
+        )
+        self._records.append(rec)
+        logger.debug("hypothesis_state: recorded id={} title={!r}", rec.id, rec.title)
```

Ruff (0.15.15)

[error] Too many arguments for `logging` format string

(PLE1205)